### DISCUSSION TITLE
How to customize tab title for blog-only site?

### DISCUSSION THREAD

**Discussion by luyizhang** [facebook/docusaurus#7934] [Q&A]
I'm using docusaurus v2 to create a blog-only site. Followed the instructions [here](https://docusaurus.io/docs/blog#:~:text=Blog%2Donly%20mode%E2%80%8B,example.com%2Fblog%2F%20). Everything's working great, but the tab title for the index blog list page is showing up as `[site title] | [site title]`.

Is there any way to customize the tab title for the index blog list page? Or, how do I get it not to repeat the site title?

**Comment by slorber** [comment]
https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog#blogTitle

afaik we don't permit to opt out for the ` | site title` suffix, there's already an issue for that

**Reply by luyizhang** [reply]
I tried changing the `blogTitle` as well, but it still was showing up the `siteTitle | siteTitle`! I'm not sure if it's possibly related to another issue I'm having [https://github.com/facebook/docusaurus/discussions/7936](url)?

**Reply by slorber** [reply]
please create a repro with docusaurus.new where we can see your problem

I can see myself that this works

![CleanShot 2022-08-11 at 10 32 35@2x](https://user-images.githubusercontent.com/749374/184094762-3540f838-75d4-444e-b2dd-ce1f213792ee.png)

**Reply by devkanro** [reply]
@slorber You must set the `routeBasePath` to '/', then the blogTitle will not work.
The title will show as `[site title] | [site title]`

**Reply by slorber** [reply]
@devkanro related to https://github.com/facebook/docusaurus/issues/5878
